lib/game.rb

```diff
@@ -0,0 +1,36 @@
+class Game
+  @@games = {}
+
+  def self.add(game)
```

Comment:
so much cleaner with variable change!

Reply:
So much more we can do to clean it up.. but that can wait until the next refactor.